cfgov/cfgov/settings/production.py

```diff
@@ -1,19 +1,34 @@
 import sys
 
 from .base import *
+from os.path import exists
 
 # log to disk when running in mod_wsgi, otherwise to console
 # This avoids permissions problems when logged in users (or CI jobs)
 # can't write to the log file.
 if sys.argv and sys.argv[0] == 'mod_wsgi':
-    default_loggers = ['disk']
+    default_loggers = ['disk', 'syslog']
 else:
-    default_loggers = ['console']
+    default_loggers = ['console', 'syslog']
+
+
+# This allows the syslog stuff to work on OS X
+syslog_device = next(l for l in ['/dev/log', '/var/run/syslog'] if exists(l))
```

Comment:
too inscrutable? Or you don't like the whole idea of checking which file exists?